propkatraj/tests/test_regression.py

```diff
@@ -52,32 +53,8 @@ def pka_from_file(filename):
     return resnums, pkas
 
 
-@pytest.mark.parametrize('selection', ['protein', 'array', 'list'])
+@pytest.mark.parametrize('selection', ['protein', 'ag', 'array', 'list'])
```

Comment:
I honestly have no idea why we are testing anything except a selection string. As far as I am concerned, I would just test "protein" and perhaps "protein and resid 1-10".

Reply:
It seems like line 79 was erroneously removed and this one (which was intended for the old method) was kept instead.

Reply:
@IAlibay I figured that I'd do the union of the two parameter sets. I'll switch it to just `protein` and `ag`